mail_tracking/models/mail_thread.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Alexandre Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, api
+from email.utils import COMMASPACE
+
+
+class MailThread(models.AbstractModel):
+    _inherit = "mail.thread"
+
+    @api.multi
+    @api.returns('self', lambda value: value.id)
+    def message_post(self, body='', subject=None, message_type='notification',
```

Comment:
Since all you care about is a `kwarg`, you can use this here:

```suggestion
    def message_post(self, *args, **kwargs):
```